app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardScreen.kt

```diff
@@ -92,69 +107,183 @@ fun KeyboardScreen(
     val hideLetters = (settings?.hideLetters ?: DEFAULT_HIDE_LETTERS).toBool()
     val hideSymbols = (settings?.hideSymbols ?: DEFAULT_HIDE_SYMBOLS).toBool()
 
-    Box(
-        contentAlignment = alignment,
-        modifier = Modifier
-            .padding(bottom = pushupSizeDp),
-    ) {
-        Column(
-            modifier = Modifier
-                .background(MaterialTheme.colorScheme.onBackground),
+    if (mode == KeyboardMode.EMOJI) {
```

Comment:
It *would* be possible to move everything inside of this block into its own composable, which might be better for organization's sake. 

I leave that up to you, and it can always be done at a later time too.